src/TLuaInterpreter.cpp

```diff
@@ -12494,7 +12494,14 @@ void TLuaInterpreter::logError( std::string & e, const QString & name, const QSt
         mpHost->mpEditorDialog->mpErrorConsole->printDebug(green, black, s2 );
         mpHost->mpEditorDialog->mpErrorConsole->printDebug(red, black, s3 );
     }
-
+    
+    if( mpHost->mpConsole && mpHost->mEchoLuaErrors )
```

Comment:
I'll probably want to have the error messages in the main window all the time, if I have that option set. Usually I am coding something and have the esitor window open all the time, which would remove a lot of utility of this option.

If you want to put another restriction to this option (which I wouldn't do), I'd rather do the opptosite and show them only when the editor is open, similar to the debug window.